### DISCUSSION TITLE
how to flash raspberry pi pico?

### DISCUSSION THREAD

**Discussion by kimgalibert** [shanteacontrols/OpenDeck#226] [General]
is the procedure to flash the raspberry pi pico similar to any of the other boards?

can you post a cabling list/guide?

thanks

**Comment by stllvdv** [comment]
Mount the PICO with the reset button pressed, delete what’s on there and drag and drop your configuration/YAML-file, disconnect and reconnect. EDIT: forget the nonsense above, you need to drop the compiled binary, not the YAML-file, of course.

**Comment by paradajz** [comment]
Here's the guide:
https://github.com/shanteacontrols/OpenDeck/wiki/Flashing-RP2040-boards
